docs/astro/src/content/docs/guide/tooling/vscode.mdx

```diff
@@ -22,32 +22,17 @@ We support many other tools and editors, see [here](https://github.com/slint-ui/
 ## Setting Up VS Code
 
 
-<Tabs syncKey="dev-platform">
-<TabItem label="Windows" icon="seti:windows">
-<Steps>
-1. Install VS Code. 
-Download it [here](https://code.visualstudio.com).
-
-2. Install the Slint extension.
-Find it [here](https://marketplace.visualstudio.com/items?itemName=Slint.slint).
 
-3. Create a new project based on a Slint template.
-This is done via the command palette (Ctrl+Shift+P).
-
-4. Choose your language.
-
-</Steps>
-</TabItem>
-<TabItem label="macOS" icon="apple">
 <Steps>
+
 1. **Install VS Code.**   
 Download it [here](https://code.visualstudio.com).
 
 2. **Install the Slint extension.**  
 Find it [here](https://marketplace.visualstudio.com/items?itemName=Slint.slint).
 
-3. **Create a new project based on a Slint template.**  
-This is done via the command palette (CMD+Shift+P).
+3. **Create a new project based on a Slint template.** 
+This is done via the command palette (CTRL+Shift+P) or on MacOS (CMD+Shift+P).
```

Comment:
Ideally it should also state what to type in the command palette, in text.
(Screenshot alone is not accessible and cannot be copy-pasted)